### ISSUE TITLE
Default values for hyperparameters

### ISSUE THREAD

**Issue by samsledje** [jszym/rapppid#1]
Hi Joseph!

Could you update the readme with recommended default parameters for things such as batch size, embedding size, number of epochs, LSTM dropout rate, etc.?

Thanks!

**Comment by jszym** [comment]
Good call! I'll transcribe the table from the paper's supplemental.

**Comment by jszym** [comment]
`docs/train.md` has been updated and I've created a `docs/hyperparams.md` file as well.

Let me know if these updates answer your questions :) 

Also be sure to let me know if you run in to any trouble running RAPPPID, I'm happy to help!
